share/picongpu/examples/Bunch/include/picongpu/param/fieldBackground.param

```diff
@@ -20,35 +20,43 @@
 #pragma once
 
 /** Load pre-defined templates */
-#include "picongpu/fields/background/templates/TWTS/TWTS.hpp"
+#ifndef TWTSFAST
```

Comment:
This will be always true. The reason is that the macro definition `TWTSFAST` is never defined somewhere in the code.
There is a define `PARAM_TWTSFAST` available.